vaadin-spring/src/main/java/com/vaadin/flow/spring/SpringServlet.java

```diff
@@ -161,6 +161,11 @@ private Properties config(Properties initParameters) {
         properties.putAll(initParameters);
         PROPERTY_NAMES.forEach(property -> setProperty(property, properties));
 
+        // this property is reversed
+        if ("true".equalsIgnoreCase(
```

Comment:
`true`  is `Boolean.TRUE` in a typesafe way.

Reply:
ok. done